src/licensedcode/data/licenses/npl-4.0.yml

```diff
@@ -0,0 +1,8 @@
+key: npl-4.0
```

Comment:
Use instead `non-violent-4.0`

Reply:
The file should also be renamed right?

Reply:
Yes the names of the .LICENSE and .yml file are also needed to be changed as per the key name

Reply:
Okay Thanks :)